packages/agent/src/agents/operators/__tests__/agentSelector.spec.ts

```
@@ -62,14 +62,41 @@ const llmErr = (msg = 'LLM error') => new Error(msg);
 function makeAgents() {
   return new Map([
     [
-      'agent1',
+      'agent1|user1',
       new MockSnakAgent('agent1', 'Handles blockchain operations') as any,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)